README.md

```diff
@@ -152,18 +151,19 @@ $ cargo build --target i686-unknown-linux-gnu
 
 ## SIMD target backends
 
-Target backend selection within `simd` must be done manually by setting the
-`RUSTFLAGS` environment variable to one of the below options:
+The SIMD target backend selection is done automatically at runtime depending
+on the available CPU features, provided the appropriate feature flag is enabled.
 
-| CPU feature | `RUSTFLAGS`                     | Requires nightly? |
-| :---        | :---                            | :---              |
-| avx2        | `-C target_feature=+avx2`       | no                |
-| avx512ifma  | `-C target_feature=+avx512ifma` | yes               |
```

Comment:
I noticed all of this documentation is removed, but the newly introduced crate features aren't equivalent and perhaps this should be documented.

Namely these flags *bypass autodetection* and force the use of SIMD features.

Reply:
yeah I can take care of documenting that - just need to re-organise a bit to make room for the vendored dep first